<!DOCTYPE html>
<html>
<head>
    <title>Dictionary</title>
</head>

<body>


    <h1>Dictionary</h1>
    <p>The purpose of this dictionary is to provide a consistent representation of common protein sequence features
        for all groups at EBI</p>
    <h2>General Guidelines</h2>

    <section title="borders">
        <h3>Borders</h3>
        <ul>
            <li>Use borders for individual features but not for groups of them as it could be confusing</li>
            <li>Use a darker colour for the border, better if it is a variation of the defined colour for the feature. In these guidelines, we use black just to make cleared that they do have a border</li>
            <li>If possible, use always straight edges. It is easy to manage different ending styles representing
                uncertanty on the start or end positions.</li>
            <li>Rounded edges are fine when start and end positions are slightly unsure,
                as it happens with domains based on signatures</li>
        </ul>

        <h3>Endings for features covering multiple amino acids</h3>
        <table border="1" cellpadding="3" cellspacing="0">
            <tr>
                <td>Known start, known end -e.g., (1,100): Straight or rounded edge, depending on your preferred edge style</td>
                <td><img alt="" src="images/dictionary/startKnown_endKnown.png"/></td>
            </tr>
            <tr>
                <td>Start is known to happen at or before certain position, known end -e.g., (at or before 8, 17):
                    Triangle pointing to the left with its base on the known start position, regular ending on the right<br/>
                    Note: The height of the triangle should be as short as possible, just enough to show the triangle shape</td>
                <td><img alt="" src="images/dictionary/startBeforeKnownPos_endKnown.png"/></td>
            </tr>
            <tr>
                <td>Known start, end is known to happen at or after certain position -e.g., (5, at or after 14):
                    Regular ending on the left, triangle pointing to the right with its base on the known end position<br/>
                    Note: The height of the triangle should be as short as possible, just enough to show the triangle shape</td>
                <td><img alt="" src="images/dictionary/startKnown_endAfterKnownPos.png"/></td>
            </tr>
            <tr>
                <td>Start is known to happen at or before certain position, end is known to happen at or after certain position -e.g.,
                    (at or before 4, at or after 14):
                    Triangle pointing to the left with its base on the known start position,
                    triangle pointing to the right with its base on the known end position<br/>
                    Note: The height of the triangle should be as short as possible, just enough to show the triangle shape</td>
                <td><img alt="" src="images/dictionary/startBeforeKnownPos_endAfterKnownPos.png"/></td>
            </tr>
            <tr>
                <td>Start is known to happen at or after certain position, known end -e.g., (at or after 8, 17):
                    Backgroud-coloured triangle pointing to the right with its base on the known start position, regular ending on the right<br/>
                    Note: The height of the triangle should be as short as possible, just enough to show the triangle shape</td>
                <td><img alt="" src="images/dictionary/startAfterKnownPos_endKnown.png"/></td>
            </tr>
            <tr>
                <td>Known start, end is known to happen at or before certain position -e.g., (4, at or before 14):
                    Regular ending on the left, background-coloured triangle pointing to the left with its base on the known end position<br/>
                    Note: The height of the triangle should be as short as possible, just enough to show the triangle shape</td>
                <td><img alt="" src="images/dictionary/startKnown_endBeforeKnownPos.png"/></td>
            </tr>
            <tr>
                <td>Start is known to happen at or after certain position, end is known to happen at or before certain position -e.g.,
                    (at or after 2, at or before 14):
                    Backgroud-coloured triangle pointing to the right with its base on the known start position,
                    background-coloured triangle pointing to the left with its base on the known end position<br/>
                    Note: The height of the triangle should be as short as possible, just enough to show the triangle shape</td>
                <td><img alt="" src="images/dictionary/startAfterKnownPos_endBeforeKnowPos.png"/></td>
            </tr>
            <tr>
                <td>Start is known to happen around (at or 1 before or 1 after) certain position, known end -e.g., (around 8, 17):
                    "S" around the known start position, regular ending on the right<br/>
                    Note: The "S" should be as narrow as possible, just enough to show the "S" shape</td>
                <td><img alt="" src="images/dictionary/startAroundKnownPos_endKnown.png"/></td>
            </tr>
            <tr>
                <td>Known start, end is known to happen around (at or 1 before or 1 after) certain position -e.g., (5, around 14):
                    Regular ending on the left, "S" around the known end position<br/>
                    Note: The "S" should be as narrow as possible, just enough to show the "S" shape</td>
                <td><img alt="" src="images/dictionary/startKnown_endAroundKnownPos.png"/></td>
            </tr>
            <tr>
                <td>Start is known to happen around (at or 1 before or 1 after) certain position,
                    end is known to happen around (at or 1 before or 1 after) certain position -e.g.,
                    (around 8, around 14):
                    "S" around the known start position,
                    "S" around the known end position<br/>
                    Note: The "S" should be as narrow as possible, just enough to show the "S" shape</td>
                <td><img alt="" src="images/dictionary/startAroundKnownPos_endAroundKnownPos.png"/></td>
            </tr>
            <tr>
                <td>Start is known to happen between two known positions n and m, known end -e.g., (between 5-8, 17):
                    Height of the rectangle between n-m reduced to 50% and aligned to the top, regular ending on the right</td>
                <td><img alt="" src="images/dictionary/startBetweenKnownPos_endKnown.png"/></td>
            </tr>
            <tr>
                <td>Known start, end is known to happen between two known positions n and m -e.g., (5, between 17-20):
                    Regular ending on the left, height of the rectangle between n-m reduced to 50%, aligned to the bottom</td>
                <td><img alt="" src="images/dictionary/startKnown_endBetweenKnownPos.png"/></td>
            </tr>
            <tr>
                <td>Start is known to happen between two known positions,
                    end is known to happen around (at or 1 before or 1 after) certain position -e.g.,
                    (between 5-8, between 17-20):
                    Height of the rectangle between n-m reduced to 50% and aligned to the top,
                    height of the rectangle between n-m reduced to 50%, aligned to the bottom</td>
                <td><img alt="" src="images/dictionary/startBetweenKnownPos_endBetweenKnownPos.png"/></td>
            </tr>
            <tr>
                <td>An envelope surrounding the feature: envelopes goes from position 4 to 13 while inner feature
                goes from position 5 to 12</td>
                <td><img alt="" src="images/dictionary/envelope.png"/></td>
            </tr>
        </table>
        <h3>One single amino acid features</h3>
        <table border="1" cellpadding="3" cellspacing="0">
            <tr>
                <td>One single amino acid feature, with several possible positions:
                    Located at the most probable position, all positions available as additional information<br/>
                    Note: The tooltip style and the informations displayed are just examples</td>
                <td><img alt="" src="images/dictionary/singleAAWithSeveralPossiblePositions.png"/></td>
            </tr>
            <tr>
                <td>One single amino acid feature, known to happen within a region, exact position uncertain:
                    Use the shape corresponding to the feature in the middle of the region, and dotted lines to both sides ending with a
                    short vertical line (not longer that rectangles height) on the starting and ending positions of the region.<br/>
                    It is also possible to omit the shape</td>
                <td><img alt="" src="images/dictionary/singleAAInRegion.png"/><br/><img alt="" src="images/dictionary/singleAAInRegion_noShape.png"/></td>
            </tr>
        </table>
        <h3>Discontinuous features</h3>
        <table border="1" cellpadding="3" cellspacing="0" width="100%">
            <tr>
                <td>Discontinuous features: Depict the partial features as usual, and connect them with a bridge-like line
                </td>
                <td><img alt="" src="images/dictionary/discontinuousFT_top.png"/></td>
            </tr>
        </table>
    </section>

    <section title="sizeAndFont">
        <h2>Zooming and scaling</h2>
        <p>How many pixels should be used to represent one amino acid when zooming to that level? How to balance the
            sequence length and the SVG width? These are some recommendations.
            <ul>
                <li>Use monospace font</li>
                <li>11 font-size is not to big nor to small</li>
                <li>Scale shapes and rectangles to the number of pixels used by one amino acid when zooming to
                    amino acid scale</li>
                <li>Do not scale shapes below 13 pixels, otherwise they will not be visible or distinguishable</li>
                <li>Do not allow shapes above 25 pixels, otherwise you will have too much space per amino acid</li>
            </ul>
        </p>
        <p>
            <span style="font-size: 12px; font-family: monospace">A R N D C E Q G H I L K M F P S T W Y V</span>
        </p>
    </section>
    <section title="cv">
        <h2>Controlled Vocabularies</h2>
        <p>Some of the data sources whose features are contained in this document use controlled
            vocavularies (CV) in order to define the type of fetures. These are the CVs currently used:
        </p>
        <ul>
            <li>Biosapiens Ontology (BS)</li>
            <li>Molecular Interaction (MI)</li>
            <li>Protein Modifications (MOD)</li>
            <li>PSI Mass Spectrometry Ontology (MS)</li>
            <li>Sequence Types and Features (SO)</li>
        </ul>
    </section>

    <section title="Categories and types">
        <h2>UniProt &amp; PDB Features</h2>
        <table border="1" cellpadding="3" cellspacing="0" width="100%">
            <tr>
                <th>Category</th>
                <th>Type</th>
                <th>Description</th>
                <th>CV id</th>
                <th>Colour</th>
                <th>Shape</th>
                <th>Comments</th>
            </tr>
            <tr>
                <th rowspan="6">Molecule Processing</th>
                <td>Chain</td>
                <td>The polypeptide sequence that remains when the cleaved peptide regions have been
                    cleaved from the immature peptide.</td>
                <td>SO:0000419</td>
                <td bgcolor="#CC9933">#CC9933, RGB: 204, 153, 51</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Transit</td>
                <td>The transit_peptide is a short region at the N-terminus of the peptide that directs the
                    protein to an organelle (chloroplast, mitochondrion, microbody or cyanelle).</td>
                <td>SO:0000725</td>
                <td bgcolor="#009966">#009966, RGB: 0, 153, 102</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Initiator methionine</td>
                <td>The initiator methionine that has been cleaved from a mature polypeptide sequence.</td>
                <td>SO:0000691</td>
                <td bgcolor="#996633">#996633, RGB: 153, 102, 51</td>
                <td>Arrow-triangle pointing down<img alt="" src="images/dictionary/methionine.png"/></td>
                <td>It is always a feature removed at position 1; however, the amino acid M is shown in the sequence </td>
            </tr>
            <tr>

                <td>Propeptide</td>
                <td>Part of a peptide chain which is cleaved off during the formation of the mature protein.</td>
                <td>SO:0001062</td>
                <td bgcolor="#99CCCC">#99CCCC, RGB: 153, 204, 204</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Peptide</td>
                <td>Active peptides are proteins which are biologically active, released from a precursor
                    molecule.</td>
                <td>SO:0001064</td>
                <td bgcolor="#006699">#006699, RGB: 0, 102, 153</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Signal peptide</td>
                <td>The signal_peptide is a short region of the peptide located at the N-terminus that directs
                    the protein to be secreted or part of membrane components.</td>
                <td>SO:0000418</td>
                <td bgcolor="#CC0033">#CC0033, RGB: 204, 0, 51</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>
                <th>Isoform</th>
                <td>Sequence variation</td>
                <td>Description of sequence variants produced by alternative splicing, alternative
                    promoter usage, alternative initiation and ribosomal frameshifting.</td>
                <td>SO:0001149</td>
                <td bgcolor="#33CC99">#33CC99, RGB: 51, 204, 153</td>
                <td>Rectangle</td>
                <td>Note: Isoforms are not included in the FTViewer</td>
            </tr>
            <tr>
                <th rowspan="5">Structural features</th>
                <td>Turn</td>
                <td>A reversal in the direction of the backbone of a protein that is stabilized by hydrogen bond between
                    backbone NH and CO groups, involving no more than 4 amino acid residues.</td>
                <td>SO:0001128</td>
                <td bgcolor="#0571AF">#0571AF, RGB: 204, 204, 204</td>
                <td>Ribbon or rectangle <img alt="" src="images/dictionary/turn.png"/></td>
                <td></td>
            </tr>
            <tr>

                <td>Beta strand</td>
                <td>A beta strand describes a single length of polypeptide chain that forms part of a beta sheet. A single continuous stretch
                    of amino acids adopting an extended conformation of hydrogen bonds between the N-O and the C=O of another part of the peptide.
                    This forms a secondary protein structure in which two or more extended polypeptide regions are hydrogen-bonded to one another
                    in a planar array.</td>
                <td>SO:0001111</td>
                <td bgcolor="#FFCC00">#FFCC00, RGB: 255, 204, 0</td>
                <td>Arrow or rectangle <img alt="" src="images/dictionary/betaStrand.png"/></td>
                <td>Jmol cartoon colors</td>
            </tr>
            <tr>

                <td>Alpha helix</td>
                <td>The helix has 3.6 residues per turn which corersponds to a translation of 1.5 angstroms (= 0.15 nm) along the helical axis.
                    Every backbone N-H group donates a hydrogen bond to the backbone C=O group of the amino acid four residues earlier.</td>
                <td>SO:0001117</td>
                <td bgcolor="#FF0066">#FF0066, RGB: 255, 0, 102</td>
                <td>Spiral or rectangle<img alt="" src="images/dictionary/alphaHelix.png"/></td>
                <td>Jmol cartoon colors</td>
            </tr>
            <tr>

                <td>Cross link</td>
                <td>Post-translationally formed amino acid bonds.</td>
                <td>MOD:00033</td>
                <td bgcolor="#FF6600">#FF6600, RGB: 255, 102, 0</td>
                <td>Either bridge over the sequence or rectangle for one position with antenna <img alt="" src="images/dictionary/crossLink.png"/></td>
                <td></td>
            </tr>
            <tr>

                <td>Disulfide bond</td>
                <td>The covalent bond between sulfur atoms that binds two peptide chains or different parts
                    of one peptide chain and is a structural determinant in many protein molecules.</td>
                <td>MOD:00689</td>
                <td bgcolor="#23B14D">#23B14D, RGB: 51, 153, 102</td>
                <td>Either bridge over the sequence or rectangle for one position with antenna <img alt="" src="images/dictionary/disulfideBond.png"/></td>
                <td></td>
            </tr>
            <tr>
                <th rowspan="13">Domains and sites</th>
                <td>Region (maybe nucleotide region)</td>
                <td>A sequence_feature with an extent greater than zero. A nucleotide region is composed of
                    bases and a polypeptide region is composed of amino acids.</td>
                <td>SO:0000001</td>
                <td bgcolor="#B33E00">#B33E00, RGB: 0, 102, 153</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Coiled coil</td>
                <td>A coiled coil is a structural motif in proteins, in which alpha-helices are coiled together
                    like the strands of a rope.</td>
                <td>SO:0001080</td>
                <td bgcolor="#006699">#006699, RGB: 0, 102, 153</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Motif</td>
                <td>A conserved motif is a short (up to 20 amino acids) region of biological interest that
                    is conserved in different proteins. They may or may not have functional or structural
                    significance within the proteins in which they are found.</td>
                <td>SO:0100017</td>
                <td bgcolor="#402060">#402060, RGB: 64, 32, 96</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Repeat</td>
                <td>A polypeptide_repeat is a single copy of an internal sequence repetition.</td>
                <td>SO:0001068</td>
                <td bgcolor="#9900FF">#9900FF, RGB: 153, 0, 255</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Calcium binding</td>
                <td>A structurally or functionally defined protein region. In proteins with multiple domains, the
                    combination of the domains determines the function of the protein. A region which has been shown to recur
                    throughout evolution.</td>
                <td>SO:0000417</td>
                <td bgcolor="#FF3399">#FF3399, RGB: 255, 51, 153</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>DNA binding</td>
                <td>A structurally or functionally defined protein region. In proteins with multiple domains, the
                    combination of the domains determines the function of the protein. A region which has been shown to recur
                    throughout evolution.</td>
                <td>SO:0000417</td>
                <td bgcolor="#009933">#009933, RGB: 0, 153, 51</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Domain</td>
                <td>A structurally or functionally defined protein region. In proteins with multiple domains, the
                    combination of the domains determines the function of the protein. A region which has been shown to recur
                    throughout evolution.</td>
                <td>SO:0000417</td>
                <td bgcolor="#9999FF">#9999FF, RGB: 153, 153, 255</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Zinc finger</td>
                <td>A structurally or functionally defined protein region. In proteins with multiple domains, the
                    combination of the domains determines the function of the protein. A region which has been shown to recur
                    throughout evolution.</td>
                <td>SO:0000417</td>
                <td bgcolor="#990066">#990066, RGB: 153, 0, 102</td>
                <td>Rectangle</td>
                <td><a href="http://en.wikipedia.org/wiki/CPK_coloring">CPK colouring for zinc</a></td>
            </tr>
            <tr>

                <td>Nuclear Phosphorylation</td>
                <td>A structurally or functionally defined protein region. In proteins with multiple domains, the
                    combination of the domains determines the function of the protein. A region which has been shown to recur
                    throughout evolution.</td>
                <td>SO:0000417</td>
                <td bgcolor="#FF9900">#FF9900, RGB: 255, 153, 0</td>
                <td>Rectangle</td>
                <td><a href="http://en.wikipedia.org/wiki/CPK_coloring">CPK colouring for phosphorus</a></td>
            </tr>
            <tr>

                <td>Metal binding</td>
                <td>A binding site that, in the polypeptide molecule, interacts selectively and
                    non-covalently with metal ions.</td>
                <td>SO:0001092</td>
                <td bgcolor="#009900">#009900, RGB: 0, 153, 0</td>
                <td>Diamond <img alt="" src="images/dictionary/metal.png"/></td>
                <td></td>
            </tr>
            <tr>

                <td>Site</td>
                <td>Biological sequence region that can be assigned to a specific subsequence of a
                    polypeptide.</td>
                <td>SO:0000839</td>
                <td bgcolor="#660033">#660033, RGB: 102, 0, 51</td>
                <td>Chevron arrow <img alt="" src="images/dictionary/site_2.png"/></td>
                <td>Usually one amino acid but it also can be two flanking amino acids</td>
            </tr>
            <tr>

                <td>Binding site</td>
                <td>(i) Binding site for any chemical group (co-enzyme, prosthetic group, etc.). <br/>
                    (ii) A polypeptide binding motif is a short (up to 20 amino acids) polypeptide region of
                    biological interest that contains one or more amino acids experimentally shown to bind to a
                    ligand.</td>
                <td>(i) SO:0001091, (ii) SO:0100018</td>
                <td bgcolor="#006699">#006699, RGB: 0, 102, 153</td>
                <td>Cat face <img alt="" src="images/dictionary/binding_2.png"/></td>
                <td></td>
            </tr>
            <tr>

                <td>Active site</td>
                <td>Amino acid involved in the activity of an enzyme.</td>
                <td>SO:0001104</td>
                <td bgcolor="#FF6666">#FF6666, RGB: 255, 102, 102</td>
                <td>Circle <img alt="" src="images/dictionary/activeSite.png"/></td>
                <td></td>
            </tr>
            <tr>
                <th rowspan="3">Post translational modifications</th>
                <td>Modified residue</td>
                <td>A region where a transformation occurs in a protein after it has been synthesized.
                    This which may regulate, stabilize, crosslink or introduce new chemical functionalities in
                    the protein.</td>
                <td>SO:0001089</td>
                <td bgcolor="#000066">#000066, RGB: 0, 0, 102</td>
                <td>Triangle <img alt="" src="images/dictionary/modifiedResidue.png"/></td>
                <td></td>
            </tr>
            <tr>

                <td>Lipid</td>
                <td>A protein modification that effectively results from forming an adduct with a compound
                    containing a lipid-like group either through acylation, alkylation, or amidation.</td>
                <td>MOD:01155</td>
                <td bgcolor="#99CC33">#99CC33, RGB: 153, 204, 51</td>
                <td>Wave <img alt="" src="images/dictionary/lipid.png"/></td>
                <td></td>
            </tr>
            <tr>

                <td>Glycosylation</td>
                <td>A protein modification that effectively replaces a hydrogen atom with a
                    carbohydrate-like group through a glycosidic bond.</td>
                <td>MOD:00693</td>
                <td>
                    Without distinguishing the type of linkage: <span style="background-color:#CC3366;">#CC3366, RGB: 204, 51, 102</span><br/>
                    C-linked: <span style="background-color:#D3D3D3;">#D3D3D3</span><br/>
                    N-linked: <span style="background-color:#87CEE6;">#87CEE6</span><br/>
                    O-linked: <span style="background-color:#FF0000;">#FF0000</span>
                </td>
                <td>Elongated hexagon <img alt="" src="images/dictionary/carbohydration-glycosylation.png" /></td>
                <td><a href="http://en.wikipedia.org/wiki/CPK_coloring">Linkage colours</a></td>
            </tr>
            <tr>
                <th>Variations</th>
                <td>Variant</td>
                <td>Describes the natural sequence variants due to polymorphisms, disease-associated
                    mutations, RNA editing and variations between strains, isolates or cultivars.</td>
                <td>SO:0001147</td>
                <td bgcolor="#006666">Heatmap. If only one colour is used then #006666 TGB:0, 102, 102</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>
                <th rowspan="7">Sequence information</th>
                <td>Compositional bias</td>
                <td>Polypeptide region that is rich in a particular amino acid or homopolymeric and greater than
                    three residues in length.</td>
                <td>SO:0001066</td>
                <td bgcolor="#FF3366">#FF3366, RGB: 255, 51, 102</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Mutagen</td>
                <td>Site which has been experimentally altered.</td>
                <td>SO:0001148</td>
                <td bgcolor="#FF9900">#FF9900, RGB: 255, 153, 0</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Conflict</td>
                <td>Different sources report differing sequences.</td>
                <td>SO:0001085</td>
                <td bgcolor="#6633CC">#6633CC, RGB: 102, 51, 204</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Non consecutive</td>
                <td>Indicates that two consecutive residues in a fragment sequence are not consecutive in
                    the full-length protein and that there are a number of unsequenced residues between them.</td>
                <td>SO:0001083</td>
                <td bgcolor="#FF0033">#FF0033, RGB: 255, 0, 51</td>
                <td>// representing a break in the sequence<img alt="" src="images/dictionary/nonConsecutive.png" /></td>
                <td></td>
            </tr>
            <tr>

                <td>Non terminal residue</td>
                <td>The residue at an extremity of the sequence is not the terminal residue.</td>
                <td>SO:0001084</td>
                <td bgcolor="#339933">#339933, RGB: 51, 153, 51</td>
                <td>// either at the beginning or end of the sequence representing uncertainty about the limits<img alt="" src="images/dictionary/nonTerminal.png" /></td>
                <td></td>
            </tr>
            <tr>

                <td>Unsure, seq. uncertainty</td>
                <td>Describes the positions in a sequence where the authors are unsure about the sequence
                    assignment.</td>
                <td>SO:0001086</td>
                <td bgcolor="#33FF00">#33FF00, RGB: 51, 255, 0</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Non standard amino acid</td>
                <td>A protein modification that inserts or replaces a residue with a natural, non-standard
                    encoded residue, such as N-formyl-L-methionine, L-selenocysteine, or L-pyrrolysine.</td>
                <td>MOD:00868, MOD:01187 (L-pyrrolysine), MOD:00031 (L-selenocysteine),
                    MOD:00030 (N-formyl-L-methionine)</td>
                <td bgcolor="#330066">#330066, RGB: 51, 0, 102</td>
                <td>Pentagon pointing down <img alt="" src="images/dictionary/nonStandard_2.png"/></td>
                <td></td>
            </tr>
            <tr>
                <th rowspan="3">Topology</th>
                <td>Topological domain, extramembrane</td>
                <td>Polypeptide region that is localized outside of a lipid bilayer.</td>
                <td>SO:0001072</td>
                <td bgcolor="#CC0000">#CC0000, RGB: 204, 0, 0</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Intramembrane</td>
                <td>Polypeptide region present in the lipid bilayer.</td>
                <td>SO:0001075</td>
                <td bgcolor="#0000CC">#0000CC, RGB: 0, 0, 204</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>

                <td>Transmembrane</td>
                <td>Polypeptide region traversing the lipid bilayer.</td>
                <td>SO:0001077</td>
                <td bgcolor="#CC00CC">#CC00CC, RGB: 204, 0, 204</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
        </table>
        <h2>InterPro</h2>
        <span style="font-weight: bold">Category: </span>InterPro Domains<br/>
        <span style="font-weight: bold">Type: </span>About 7181 InterPro domains are available, a comprenhensive list is available at
        <a href="https://code.google.com/p/interproscan/source/browse/trunk/core/web/src/main/resources/entry_colours.properties">
            https://code.google.com/p/interproscan/source/browse/trunk/core/web/src/main/resources/entry_colours.properties
        </a><br/>
        <span style="font-weight: bold">Description: </span>Domains are distinct functional, structural or sequence units that may exist in a variety
        of biological contexts. Taken from
        <a href="http://www.ebi.ac.uk/interpro/user_manual.html#faqs_04">
            http://www.ebi.ac.uk/interpro/user_manual.html#faqs_04
        </a><br/>
        <span style="font-weight: bold">CV id: </span><br/>
        <table border="1" cellpadding="3" cellspacing="0">
            <tr>
                <td>BS:00333</td>
                <td>Annotations which indicate the family to which the protein belongs</td>
            </tr>
            <tr>
                <td>SO:0100017</td>
                <td>A conserved motif is a short (up to 20 amino acids) region of biological interest that is conserved in different proteins.
                    They may or may not have functional or structural significance within the proteins in which they are found</td>
            </tr>
            <tr>
                <td>SO:0100018</td>
                <td>	A polypeptide binding motif is a short (up to 20 amino acids) polypeptide region of biological interest that contains
                    one or more amino acids experimentally shown to bind to a ligand</td>
            </tr>
            <tr>
                <td>SO:0001089</td>
                <td>A region where a transformation occurs in a protein after it has been synthesized. This which may regulate, stabilize,
                    crosslink or introduce new chemical functionalities in the protein</td>
            </tr>
            <tr>
                <td>SO:0100019</td>
                <td>A polypeptide catalytic motif is a short (up to 20 amino acids) polypeptide region that contains one or more active
                    site residues</td>
            </tr>
            <tr>
                <td>SO:0000417</td>
                <td>A structurally or functionally defined protein
                    region. In proteins with multiple domains, the combination of the domains determines the function of the
                    protein. A region which has been shown to recur throughout evolution</td>
            </tr>
            <tr>
                <td>SO:0001068</td>
                <td>A polypeptide_repeat is a single copy of an internal sequence repetition</td>
            </tr>
            <tr>
                <td>SO:0000001</td>
                <td>A sequence_feature with an extent greater than zero. A nucleotide region is
                    composed of bases and a polypeptide region is composed of amino acids</td>
            </tr>
        </table>
        <span style="font-weight: bold">Colour: </span>Colours depend on the feature itself, not on the type
        of the feature as all of them are InterPro Domains<br/>
        InterPro domains are grouped so each group should be display in a different colour; the groups are available at
        <a href="https://code.google.com/p/interproscan/source/browse/trunk/core/web/src/main/resources/entry_colours.properties">
            https://code.google.com/p/interproscan/source/browse/trunk/core/web/src/main/resources/entry_colours.properties
        </a><br/>
        Colours for each group are available as a CSS at
        <a href="https://code.google.com/p/interproscan/source/browse/trunk/core/web/src/main/webapp/resources/css/domain.css">
            https://code.google.com/p/interproscan/source/browse/trunk/core/web/src/main/webapp/resources/css/domain.css
        </a><br/>
        <span style="font-weight: bold">Shape: </span>Rounded-edge rectangle<br/>
        <br/>
        In this table we present the different colours used in InterPro, as modifications can happen, please
        refer to the links provided for the latest colours and features.
        <table border="1" cellpadding="3" cellspacing="0" width="100%">
            <tr>
                <td bgcolor="#b852d4">#b852d4</td>
                <td bgcolor="#00baf0">#00baf0</td>
                <td bgcolor="#ff0000">#ff0000</td>
                <td bgcolor="#33cc33">#33cc33</td>
                <td bgcolor="#ffde00">#ffde00</td>
                <td bgcolor="#ffa99e">#ffa99e</td>
                <td bgcolor="#c571ff">#c571ff</td>
                <td bgcolor="#ff9000">#ff9000</td>
                <td bgcolor="#c69c6d">#c69c6d</td>
                <td bgcolor="#ededed">#ededed</td>
            </tr>
            <tr>
                <td bgcolor="#1e82d4">#1e82d4</td>
                <td bgcolor="#a1e42d">#a1e42d</td>
                <td bgcolor="#ffff00">#ffff00</td>
                <td bgcolor="#cf161d">#cf161d</td>
                <td bgcolor="#ff6000">#ff6000</td>
                <td bgcolor="#1cbbb4">#1cbbb4</td>
                <td bgcolor="#e354e3">#e354e3</td>
                <td bgcolor="#cbcbcb">#cbcbcb</td>
                <td bgcolor="#92acd1">#92acd1</td>
                <td bgcolor="#a66c32">#a66c32</td>
            </tr>
            <tr>
                <td bgcolor="#0000ff">#0000ff</td>
                <td bgcolor="#0ff7fa">#0ff7fa</td>
                <td bgcolor="#ecd66d">#ecd66d</td>
                <td bgcolor="#ffa8fd">#ffa8fd</td>
                <td bgcolor="#9306ff">#9306ff</td>
                <td bgcolor="#998675">#998675</td>
                <td bgcolor="#c3b8f0">#c3b8f0</td>
                <td bgcolor="#7db7ff">#7db7ff</td>
                <td bgcolor="#00984a">#00984a</td>
                <td bgcolor="#bedbfe">#bedbfe</td>
            </tr>
            <tr>
                <td bgcolor="#ffdfdf">#ffdfdf</td>
                <td bgcolor="#cde6a8">#cde6a8</td>
                <td bgcolor="#fffdcc">#fffdcc</td>
                <td bgcolor="#e6e4a8">#e6e4a8</td>
                <td bgcolor="#d5a8e6">#d5a8e6</td>
                <td bgcolor="#e6c4a8">#e6c4a8</td>
                <td bgcolor="#82ca9c">#82ca9c</td>
                <td bgcolor="#dfffdc">#dfffdc</td>
                <td bgcolor="#c3b3a1">#c3b3a1</td>
                <td bgcolor="#e7e1ff">#e7e1ff</td>
            </tr>
            <tr>
                <td bgcolor="#950e0e">#950e0e</td>
                <td bgcolor="#005e20">#005e20</td>
                <td bgcolor="#a2a420">#a2a420</td>
                <td bgcolor="#e8ffa2">#e8ffa2</td>
                <td bgcolor="#ff00ff">#ff00ff</td>
                <td bgcolor="#604036">#604036</td>
                <td bgcolor="#717171">#717171</td>
                <td bgcolor="#6af0a6">#6af0a6</td>
                <td bgcolor="#5d8ca2">#5d8ca2</td>
                <td bgcolor="#6e2e9d">#6e2e9d</td>
            </tr>
            <tr>
                <td bgcolor="#e56251">#e56251</td>
                <td bgcolor="#a6ff46">#a6ff46</td>
                <td bgcolor="#bf6b66">#bf6b66</td>
                <td bgcolor="#6d4afe">#6d4afe</td>
                <td bgcolor="#a186be">#a186be</td>
                <td bgcolor="#c5209c">#c5209c</td>
                <td bgcolor="#d28428">#d28428</td>
                <td bgcolor="#86a185">#86a185</td>
                <td bgcolor="#ffc000">#ffc000</td>
                <td bgcolor="#90deda">#90deda</td>
            </tr>
            <tr>
                <td bgcolor="#ffddb2">#ffddb2</td>
            </tr>
        </table>
        <h2>IntAct</h2>
        <p>IntAct has a variety of features but for visualization purposes they have selected just some of them.</p>
        <table border="1" cellpadding="3" cellspacing="0" width="100%">
            <tr>
                <th>Category</th>
                <th>Type</th>
                <th>Description</th>
                <th>CV id</th>
                <th>Colour</th>
                <th>Shape</th>
                <th>Comments</th>
            </tr>
            <tr>
                <td>Interactions</td>
                <td>PTM</td>
                <td>A post-translational modification that elicits an allosteric response upon addition to a
                    target molecule. An allosteric post-translational modification is identified by referring to
                    its feature id.</td>
                <td>MI:1175</td>
                <td bgcolor="#AAAAAA">#AAAAAA</td>
                <td>Triangle <img alt="" src="IntAct_ptm.png"/></td>
                <td>PTMs in IntAct are obtained from Mass Spectrometry experiments reported in the literature
                    (IntAct is a curated database)</td>
            </tr>
            <tr>
                <td>Interactions</td>
                <td>Binding-associated region</td>
                <td>A region of a molecule identified as being involved in an interaction. This may or may not
                    be a region of the protein in direct contact with the interacting partner.</td>
                <td>MI:0117 + children</td>
                <td bgcolor="#FAB875">#FAB875</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>
                <td>Interactions</td>
                <td>Mutation</td>
                <td>A natural change in a sequence or structure in comparison to a reference entity due to
                    an insertion, deletion or substitution event.</td>
                <td>MI:0118</td>
                <td bgcolor="#EEEEEE">#EEEEEE</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>
                <td>Interactions</td>
                <td>Identified Peptides</td>
                <td>Peptide whose sequence is experimentally identified and can lead to a full protein
                    identification.</td>
                <td>MI:0656</td>
                <td bgcolor="#FF2600">#FF2600</td>
                <td>Rectangle</td>
                <td>In IntAct, identified peptides are obtained from Mass Spectrometry experiments
                    reported in the literature (IntAct is a curated database)</td>
            </tr>
            <tr>
                <td>Interactions</td>
                <td>Tags</td>
                <td>Molecules are often altered, or tagged, by additions to their basic structure or sequence, to aid in identification or purification.
                    This may include changes at the atomic level such as replacement of individual atoms with a radio- or heavy-isotope. </td>
                <td>Some children of MI:0505 --> MI:0373 + children, MI:0856 + children, MI:0845 + children, MI:0507 + children, MI:0950</td>
                <td bgcolor="#99CCFF">#99CCFF</td>
                <td>Rectangle</td>
                <td></td>
            </tr>
            <tr>
                <td>Interactions</td>
                <td>Isotope label</td>
                <td>One of several nuclides having the same number of protons in their nuclei and hence having
                    the same atomic number, but differing in the number of neutrons and therefore, in the mass
                    number.</td>
                <td>MI:0253 + children</td>
                <td bgcolor="#F2F200">#F2F200</td>
                <td>Star <img alt="" src="images/dictionary/IntAct_isotope.png"/></td>
                <td></td>
            </tr>
        </table>
        <h2>Pride</h2>
        <table border="1" cellpadding="3" cellspacing="0" width="100%">
            <tr>
                <th>Category</th>
                <th>Type</th>
                <th>Description</th>
                <th>CV id</th>
                <th>Colour</th>
                <th>Shape</th>
                <th>Comments</th>
            </tr>
            <tr>
                <td>-</td>
                <td>Summary of identified peptides</td>
                <td>Short sequences used in MS proteomics to identify a protein</td>
                <td>MS:1000860</td>
                <td>To be decided</td>
                <td>Rectangle without border following a saturation sytle, the darker the more peptides</td>
                <td>Peptides in Pride are obtained from Mass Spectrometry experiments directly reported to Pride</td>
            </tr>
            <tr>
                <td>-</td>
                <td>Identified peptides</td>
                <td>Short sequences used in MS proteomics to identify a protein</td>
                <td>MS:1000860</td>
                <td>To be decided</td>
                <td>Rectangle</td>
                <td>Peptides in Pride are obtained from Mass Spectrometry experiments directly reported to Pride</td>
            </tr>
            <tr>
                <td>-</td>
                <td>PTM</td>
                <td>Post-translational modification related to the MS proteomics setup</td>
                <td>MOD:000000 + children</td>
                <td>To be decided</td>
                <td>Triangle <img alt="" src="IntAct_ptm.png"/></td>
                <td>PTMs in Pride are obtained from Mass Spectrometry experiments directly reported to Pride</td>
            </tr>
        </table>
    </section>

</body>
</html>